examples/stores/counter.js

```diff
@@ -1,12 +1,12 @@
 import { INCREMENT_COUNTER, DECREMENT_COUNTER } from '../constants/ActionTypes';
 
-export default function counterStore(counter = 0, action) {
+export default function counter(state = 0, action) {
```

Comment:
Inspired by @kevinrobinson's tweets, I dropped the damned `Store` suffix. Let's just call it `counter`!